docs/gettingstarted/getting-started-splunk-setup.md

```diff
@@ -1,49 +1,48 @@
-
-  In either case, it is _strongly_ recommended that SC4S traffic be sent to HEC endpoints configured directly on the indexers rather than
-an intermediate tier of HWFs.  
-- Create a HEC token that will be used by SC4S and ensure the token has access to place events in main, _metrics, and all indexes used as
-event destinations.
-
-* NOTE: It is recommended that the "Selected Indexes" on the token configuration page be left blank so that the token has access to
-_all_ indexes, including the `lastChanceIndex`.  If this list is populated, extreme care must be taken to keep it up to date, as an attempt to
-send data to an index not in this list will result in a `400` error from the HEC endpoint. Furthermore, the `lastChanceIndex` will _not_ be
-consulted in the event the index specified in the event is not configured on Splunk.  Keep in mind just _one_ bad message will "taint" the
-whole batch (by default 1000 events) and prevent the entire batch from being sent to Splunk.
-* In case you are not using TLS on SC4S- turn off SSL on global settings for HEC in Splunk.
-- Refer to [Splunk Cloud](http://docs.splunk.com/Documentation/Splunk/7.3.1/Data/UsetheHTTPEventCollector#Configure_HTTP_Event_Collector_on_managed_Splunk_Cloud)
+1. Set up your Splunk HTTP Event Collector with the HEC endpoints behind a load balancer round-robin configuration. If you do not have a load balancer in your configuration, you can configure a list of HEC endpoint URLs with native syslog-ng load balancing.
+
+  Note that an external load balancer simplifies long-term
+maintenance, but if a load balancer is not
+available, native load balancing can be used with ten or fewer indexers where HEC is used exclusively for syslog.
+
+  SC4S traffic must be sent to HEC endpoints configured directly on the indexers rather than
+an intermediate tier of heavy forwarders.  
+
+2. Create a HEC token to be used by SC4S. Make sure the token has permissions to add events to 'main', '_metrics', and all
+event destinations indexes.
+
+3. You can leave "Selected Indexes" blank on the token configuration page so that the token has access to
+all indexes, including the `lastChanceIndex`.  If this list is populated, extreme care must be taken to keep it up to date, as an attempt to
+send data to an index that is not in this list results in a `400` error from the HEC endpoint and the `lastChanceIndex` will not be
+consulted if the index specified in the event is not configured on Splunk.  Note that one bad message can prevent and entire batch from being sent to Splunk.
+
+4. If you are not using TLS on SC4S, turn off SSL on global settings for HEC in Splunk. See [Splunk Cloud](http://docs.splunk.com/Documentation/Splunk/7.3.1/Data/UsetheHTTPEventCollector#Configure_HTTP_Event_Collector_on_managed_Splunk_Cloud)
 or [Splunk Enterprise](http://dev.splunk.com/view/event-collector/SP-CAAAE6Q) for specific HEC configuration instructions based on your
```

Comment:
is this valid url now ?

Reply:
It seems to work for me, unless you want me to direct it to a different location?